test/functional/p2p_tx_download.py

```diff
@@ -43,13 +46,10 @@ def on_getdata(self, message):
 
 
 # Constants from net_processing
-GETDATA_TX_INTERVAL = 60  # seconds
 INBOUND_PEER_TX_DELAY = 2  # seconds
-TXID_RELAY_DELAY = 2 # seconds
 OVERLOADED_PEER_DELAY = 2 # seconds
 MAX_GETDATA_IN_FLIGHT = 100
 MAX_PEER_TX_ANNOUNCEMENTS = 5000
```

Comment:
Maybe it's worth migrating some of these to `p2p.py` as well?

Reply:
I thought about this, but some of these are only used in this test.

I just realized that some of them were even duplicated amongst themselves. For instance, `INBOUND_PEER_TX_DELAY` and `NONPREF_PEER_TX_DELAY` refer to the same concept.

I'll do a second pass to further trim this